invenio_communities/templates/semantic-ui/invenio_notifications/subcommunity-invitation-request.create.jinja

```diff
@@ -0,0 +1,63 @@
+{% set request = notification.context.request %}
+{% set parent_community = request.created_by %}
+{% set community = request.receiver %}
+{% set ui = config.SITE_UI_URL %}
+{% set msg_ctx = {
+    "community_id": community.slug,
+    "community_title": community.metadata.title,
+    "parent_community_id": parent_community.slug,
+    "parent_community_title": parent_community.metadata.title,
+    "request_id": request.id,
+    "expires_at": request.expires_at | from_isodatetime | dateformat(format="long"),
+    "ui": ui,
+} %}
+
+{# TODO: use request.links.self_html when issue issue is resolved: https://github.com/inveniosoftware/invenio-rdm-records/issues/1327 #}
+{% set request_link = "{ui}/communities/{community_id}/requests/{request_id}".format(**msg_ctx) %}
+{% set account_settings_link = "{ui}/account/settings/notifications".format(**msg_ctx) %}
+
+{%- block subject -%}
+{{ _('📬 Invitation to join the {parent_community_title} community').format(**msg_ctx) }}
+{%- endblock subject -%}
+
+{%- block html_body -%}
+<table style="font-family:'Lato',Helvetica,Arial,sans-serif;border-spacing:15px">
+    <tr>
+        <td>{{ _('We would like to invite your community "{community_title}" to join as a subcommunity of "{parent_community_title}"').format(**msg_ctx) }}
+        </td>
+    </tr>
+    <tr>
+        <td><a href="{{ request_link }}" class="button">{{ _("View the request")}}</a></td>
+    </tr>
+    <tr>
+        <td>{{ _('This invitation will expire {expires_at}.').format(**msg_ctx) }}</td>
```

Comment:
```suggestion
        <td>{{ _('This invitation will expire at {expires_at}.').format(**msg_ctx) }}</td>
```

nit: since I imagine it will say "This invitation will expire **at** Tue 26 Nov"?